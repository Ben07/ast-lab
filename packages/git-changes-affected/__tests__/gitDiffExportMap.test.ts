import getDiffExportMap from '../src';

describe('getDiffExportMap', () => {
  it('can work', async () => {
    const res = await getDiffExportMap(
      '621d397900840e4aeb5cc742490431dc24a6f8a3',
      {
        transform() {
          return '';
        },
      }
    );
    expect(res).toMatchSnapshot();
  });
});

