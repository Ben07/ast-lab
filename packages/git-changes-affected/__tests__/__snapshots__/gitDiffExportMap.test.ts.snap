// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getDiffExportMap can work 1`] = `
Array [
  "/Users/jennie.ji/Projects/affected/.eslintrc.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/jest.config.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/constants.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/es6Detect/depsFromExportAll.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/es6Detect/depsFromExportNamed.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/es6Detect/exportSpecifier2Dependents.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/es6Detect/getExports.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/es6Detect/getPatternIdentifiers.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/es6Detect/importSpecifier2Dependents.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/es6Detect/index.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/filterDependents.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/index.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/isCore.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/isRealFile.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/isThirdParty.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/markDependents.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/markExports.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/replaceAlias.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/resolveModulePath.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/src/types/index.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/cant-resolve/broken.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/deepAst/funcUsePrivates.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/deepAst/funcUsePrivates2.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/deepAst/valueUsePrivates.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/deepAst/valueUsePrivates2.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportAllFrom.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportDefault.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportDefaultAliasFrom.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportDefaultFuncAlias.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportDefaultValueAlias.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportFunc.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportFuncAlias.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportNamed.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportNamedFrom.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/exportValueAlias.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/exports/noExports.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/imports/base.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/imports/importAlias.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/imports/importAll.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/imports/importDefault.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/imports/importNamed.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/imports/noImports.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/imports/require.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/module-alias/a.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/module-alias/unrelated-folder.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/module-alias/unrelated-name.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/module-directory/a.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/__fixtures__/module-directory/b.js",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/filterDependents.test.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/getEs6Dependents.test.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/markDependents.test.ts",
  "/Users/jennie.ji/Projects/affected/packages/get-dependencies/test/markExports.test.ts",
  "/Users/jennie.ji/Projects/affected/prettier.config.js",
]
`;
